### DISCUSSION TITLE
favicon and logo-small

### DISCUSSION THREAD

**Discussion by Mr-Coquin** [moonshine-software/moonshine#937] [Ideas]
Добавить возможность через config задавать свои favicon.ico, а так же дать возможность через тот же config параметр менять анимированный логотип (logo-small.svg) на 404 странице и текст страницы.

**Comment by lee-to** [comment]
ну фавиконки меняются через публикацию вьюхи vendor/moonshine/moonshine/resources/views/layouts/shared/favicon.blade.php
там ничего нужно системе кроме фавиков нет поэтому ее можно смело переопределять и не боятся за апдейты

Тоже самое и по 404

А так в целом если очень хочется то можете предложить pr

**Reply by Mr-Coquin** [reply]
Так вроде делал но при composer update перезаписывается . Можно короткую инструкцию в 2 предложения ?

**Reply by DissNik** [reply]
скопировать файл
`vendor/moonshine/moonshine/resources/views/layouts/shared/favicon.blade.php`
вот сюда
`resources/views/vendor/moonshine/layouts/shared/favicon.blade.php`

**Comment by lee-to** [comment]
Хотя знаете лого 404 я сделаю через конфиг а вот фавики пусть останутся через переопределение вьюхи, их там может быть много и так гораздно удобнее будет

**Comment by lee-to** [comment]
https://github.com/moonshine-software/moonshine/pull/938
